packages/server/src/auth/CrossmintAuth.ts

```diff
@@ -21,6 +31,63 @@ export class CrossmintAuth {
         return new CrossmintAuth(crossmint);
     }
 
+    public getJWKSUri() {
```

Comment:
nit: Casing here is weird because of inconsistency. Either: `getJwksUri` or `getJWKSURI` plz